Fix jumping in rxvt-unicode when switching focus

onodera-punpun opened the pull request:
I don't really know *why* this works, but it works.

Before when using rxvt-unicode with openbox the following would happen:

* open a file in micro
* Scroll down 80 lines
* move cursor to another window (switching focus)
* micro would jump the "view" to line 40 for some reason

zyedidia commented:
I have no idea why this works because relocate should already be true on line 301. Could you try having it print out the value of relocate? `TermMessage(relocate)`

onodera-punpun commented:
erm sorry, I said `micro would jump the "view" to line 80 for some reason`, but I meant `micro would jump the "view" to line 40 for some reason`

onodera-punpun commented:
So I just replaced the line I added with your TermMessage line, and on pretty much every mouse relation action I do it prints `true`, this includes moving the mouse, right/middle/left clicking, scrolling up and down.

zyedidia commented:
Ok, so I don't see why setting relocate to true here would change anything.

onodera-punpun commented:
Oh, I'm really stupid, I somehow replaced `false` with `true` when pushing all these patches, setting it to false fizes the problem, like you said it is obviously true by default.
